CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Airship Titanium Module
 
+## Version 6.0.1 - July 8, 2020
```

Comment:
It should be 7.0.0 and we added a ton of things since 7.0.0. Might take a look at the diff.